types/slackdown/slackdown-tests.ts

```diff
@@ -0,0 +1,6 @@
+import * as slackdown from 'slackdown';
+// tslint:disable-next-line no-duplicate-imports
+import { parse } from 'slackdown';
```

Comment:
I don't think this import is necessarily testing anything special. Can you remove it and the tslint comment?